src/System.Text.Json/tests/JsonElementWriteTests.cs

```diff
@@ -29,6 +31,16 @@ public static void WriteNumber(bool indented)
             WriteSimpleValue(indented, "42");
         }
 
+        [Theory]
+        [InlineData("12E-3")]
+        [InlineData("12e+3")]
+        //        [InlineData("1.2e+13")] Enable after https://github.com/dotnet/corefx/issues/39139 resolved
+        //        [InlineData("1.2E-3")]
+        public static void WriteScientificNumber(string value)
```

Comment:
39139 is resolved now, but I think that this method is redundant with tests added via the PR (#39406).  I'm guessing there's a merge conflict at this point now, and the easiest resolution is probably to just remove this method.

Reply:
checked the PR, it didn't covered test for capital E, i think will keep the test

Reply:
Can you merge these cases into the existing theory, then?

Reply:
sure